### PR TITLE
Removes the one slot limit for physicker

### PR DESCRIPTION
**Pull request by Plaguewalker**
## About The Pull Request
An up to date version of the same PR from last time #176 . Which removes the line of code for physicker which I am still under the impression the code to remove the class from your queue pool will be gone upon death.

## Why It's Good For The Game
It brings it in line with the other towner pilgrim roles which aren't limited to a set number of slots. Once the player dies the role is already removed from the pool list of selectable classes for the player upon respawn for pilgrim. The role in itself puts fourth the person being that of a traveling doctor akin to how the cleric is being a traveling healer as well. It should not be limited to that of one person, I understand the limiter but this will prevent the role being squandered and lost to an afk player.

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

File: code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/physicker.dm [hunk 1]
```diff
@@ -16,7 +16,6 @@
     )
     outfit = /datum/outfit/job/roguetown/adventurer/physicker
     category_tags = list(CTAG_PILGRIM)
-    maximum_possible_slots = 1
```
Comment (ThePainkiller) [inline code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/physicker.dm:19]: Up the limit to two. We don't want a literal healing party.

Comment (Plaguewalker) [reply code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/physicker.dm:19]: I'm willing to barter to three with a minor point for that being for clerics and church players have been sporadic. With three physickers and one feldsher it'll cover the major bases

Comment (ThePainkiller) [reply code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/physicker.dm:19]: My man, the number 1 being upped to 2 is all the bartering you'll get out of me. This is not a role intended to be populous. Two Pshysickers and one Feldsher are more than enough at the level of pop we manage right now, specially if they work together.

**Review by ThePainkiller — APPROVED** [review]
Two slots for travelling doctors is fine.

**Comment by Plaguewalker** [comment]
@Stutternov

**Review by Stutternov — APPROVED** [review]

**Pull request merged**